src/test/common/steps/navigation.js

```diff
@@ -175,3 +175,11 @@ When(/^I click the Cookies link$/, async function () {
 When(/^I navigate to the (.*) page$/, async function (page) {
   await navigateToPage(page)
 })
+
+When(/^I go directly to the send code page$/, async function () {
```

Comment:
You don't need to write these steps, you can use the `I navigate to the XXXX page` and as long as you've added the page to the list in `pages.js` and the page name matches what you have in your `getPageName()` function matching this then it'll all work for you.

Reply:
That won't work as that will go to the page and wait for that page to load, which it never will as we're being redirected to a different page.

Reply:
As discussed, updated to use openDirect.